src/python/bot/fuzzers/libfuzzer_engine.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2019 Google LLC
```

Comment:
This file looks wrong, you already have libFuzzer/engine.py.

Reply:
Removed!